cpp/src/Slice/Grammar.cpp

```diff
@@ -2158,7 +2158,7 @@ YYLTYPE yylloc = yyloc_default;
 {
     auto scoped = dynamic_pointer_cast<StringTok>(yyvsp[0]);
     ContainerPtr cont = currentUnit->currentContainer();
-    ContainedPtr contained = cont->lookupException(scoped->v);
+    ContainedPtr contained = cont->lookupException(scoped->v, true);
```

Comment:
This function used to have a default value of `true`.
All I did was remove the default, and write `true` here instead.
So that the function declaration still fit on one line.